Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -305,29 +354,37 @@ class SentrySessionReplay: NSObject {
     }
     
     private func takeScreenshot() {
-        guard let rootView = rootView, !processingScreenshot else { return }
+        SentryLog.debug("[Session Replay] Taking screenshot")
```

Comment:
`m`: This could be below the guard statements, otherwise you get two log message when not taking a screenshot. Basically 
```
[Session Replay] Taking screenshot
[Session Replay] Not taking screenshot, reason: root view is nil or processing screenshot
```
Which is a bit confusing and verbose.

Or you change the log message to entering taking screenshot or something like that.